docs/core/howto/tutorial/listings/finger/finger19a.tac

```diff
@@ -199,10 +191,8 @@ class UserStatusXR(xmlrpc.XMLRPC):
     def xmlrpc_getUser(self, user):
         return self.service.getUser(user)
 
+@implementer([IFingerService, IFingerSetterService])
```

Comment:
The original implements() had it as a list, but I don't think that is right.
Fixed.